ton_sdk/src/local_tvm.rs

```diff
@@ -14,29 +14,38 @@
 
 use crate::error::*;
 use std::sync::Arc;
-use chrono::prelude::*;
-use std::convert::TryFrom;
 use ton_vm::executor::Engine;
 use ton_block::{
     Message,
     Serializable,
     Deserializable,
+    MsgAddressInt
 };
 use ton_block::error::*;
 use ton_vm::stack::{CellData, IntegerData, SaveList, SliceData, Stack, StackItem};
+use ton_types::HashmapE;
 use ton_vm::SmartContractInfo;
+use ton_vm::executor::gas::gas_state::Gas;
 
 #[cfg(test)]
 #[path = "tests/test_local_tvm.rs"]
 mod tests;
 
 #[allow(dead_code)]
-pub fn local_contract_call(code: Arc<CellData>, data: Option<Arc<CellData>>, msg: &Message)
--> SdkResult<Vec<Message>> {
+pub fn local_contract_call(
```

Comment:
Too many parameters. I suggest to introduce struct to represent parameters.

Reply:
It is not a good idea to pack parameters into struct just to decrease function header. Struct should represent some logical object but these parameters are independent values